bin/trial

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
```

Comment:
I merged to hastily.
The MIT license requires that it's included with the redistributed software.
Shall we copy its text here as a comment?